Privacy update (#1018)
huggingface/chat-ui: PRIVACY.md

## Privacy

> Last updated: April 15, 2024

Users of HuggingChat are authenticated through their HF user account.

> We endorse Privacy by Design. As such, your conversations are private to you and will not shared with anyone, including model authors, and including for research or model training purposes.

Reviewer: ```suggestion
We endorse Privacy by Design. As such, your conversations are private to you and will not be shared with anyone, including model authors, for any purpose.
```
a bit clearer?

Author: i'll tweak but merging your suggestion

Author: 6bb7e17b461fda5dbfa80d49ab1308f58e483142
